docs/data/base/components/button/button.md

```diff
@@ -10,8 +12,39 @@ githubLabel: 'component: button'
 
 ## Basic usage
 
+```js
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+```
+
+{{"demo": "UnstyledButtonsSimple.js"}}
+
 ## Customizing the root element
 
+By default, the `ButtonUnstyled` renders a native `button` element.
+You are free to override this by setting the `component` or `components.Root` prop.
+If a non-interactive element (such as a span) is provided this way, the `ButtonUnstyled` will take care of adding accessibility attributes.
+
+{{"demo": "UnstyledButtonsSpan.js"}}
+
+Compare the attributes on the span with the button from the previous demo.
```

Comment:
```suggestion
Compare the attributes on the `span` with the `button` from the previous demo.
```

Reply:
Also, should this sentence go above the demo? I think it's preferable to tell the reader what to look for in the demo before they see it.